hosting_docs/source/changelog/0063-backfill_sms_event_data_for_api_performance.md

```diff
@@ -0,0 +1,34 @@
+<!--THIS FILE IS AUTOGENERATED: DO NOT EDIT-->
+<!--See https://github.com/dimagi/commcare-cloud/blob/master/changelog/README.md for instructions-->
+# 63. Backfill SMS event data for API performance
+
+**Date:** 2022-11-11
+
+**Optional per env:** _only required on some environments_
+
+
+## CommCare Version Dependency
+The following version of CommCare must be deployed before rolling out this change:
+[896676ba](https://github.com/dimagi/commcare-hq/commit/896676ba3c3b7f5e2898f055ace3ee1b196a7fac)
+
+
+## Change Context
+Backfill data in the `sms_smssubevent` table to improve the performance of the 'messaging-event' API.
+
+This operation is only required if your environment is using SMS or Email workflows.
```

Comment:
Could add something about the simple case where this migration will be run automatically on DBs below 200k rows - like that the deploy should let you know whether or not you need to run this.